articles/integrations/aws-eventbridge.md

```diff
@@ -20,29 +20,53 @@ To send Auth0 events to AWS EventBridge, you will need:
+
+2. Paste the **Event Source Name** in the event source search box to find the newly created AWS Event Source and click on it to associate it with an Event Bus.
+``Note: The AWS Event Source will remain in pending state until it gets associated with an Event Bus and all the events sent to that Event Source will be dropped.``
+
+3. Once you click in the Event Source, click on the `Associate with Event Bus` button.
+
+4. Name the AWS Event Bus the same name as the AWS Event Source. At this point, you can specify permission for this Event Bus or simply associate it.
 
-Return to AWS to complete the final steps of the integration.
 
 ## Create your rules
 
-At this point, the events that you send will be made available on your event bus. However, before you can use the data you send to EventBridge, you will need to [create rules](https://docs.aws.amazon.com/eventbridge/latest/userguide/create-event-bus.html) that map those events to event bridge targets.
+At this point, the events that you send will be made available on your event bus. However, before you can use the data you send to AWS services, you will need to [create rules](https://docs.aws.amazon.com/eventbridge/latest/userguide/create-event-bus.html) that map those events to specific targets.
 
 EventBridge uses rules, which are definitions specifying how you want incoming events routed to the desired targets. 
 
 Targets are the services, such as AWS EC2 instances, Lambda functions, Kinesis streams, or ECS tasks, that processes the event-driven data that it receives. The data received by targets are JSON-formatted.
 
 A single rule can route to one or more targets (if there are more than one, AWS processes all in parallel).
 
+To create a rule:
+
+1. First, go to the [EventBridge page](https://console.aws.amazon.com/events/home?region=us-east-1#/) and click on `Create rule`
+
+2. Provide the name of the AWS Event Bus and specify your targets
+
 ## Testing
 
-At this point, your EventBridge workflow should be fully complete. As soon as Auth0 writes the next tenant log, you should be able to go to your Target (as defined in your EventBridge rule) to find a copy of the log Auth0 has written in JSON format.
\ No newline at end of file
+At this point, your EventBridge workflow should be fully complete. As soon as Auth0 writes the next tenant log, you should be able to go to your Target (as defined in your EventBridge rule) to find a copy of the log Auth0 has written in JSON format.
+
+## Delivery attempts and retries
+
+Auth0 event are delivered to AWS via a streaming mechanism that sends each event individually as it is triggered in our system. When EventBridge is unable to receive the event we will retry to deliver the event up to three times in which case we will log the failure to deliver in our system.
```

Comment:
```suggestion
Auth0 event are delivered to AWS via a streaming mechanism that sends each event as it is triggered in our system. If EventBridge is unable to receive the event, we will retry up to three times to deliver the event, otherwise, we will log the failure to deliver in our system.
```